devel/dparser/Portfile

```diff
@@ -19,22 +20,18 @@ long_description \
     just well formed computer languages and data files, but just about any \
     wacky situation that occurs in the real world.
 
-homepage            http://dparser.sourceforge.net/
-master_sites        sourceforge
-checksums           rmd160  5075cede706ee98c4c7bec8288f4ef919bf1dc97 \
-                    sha256  4c4a61bd5954d75da89cbcfe8f6aac3a113769939b31ce2bf4b20b7b3a31cdd8
-distname            d-${version}-src
+homepage            http://github.com/jplevyak/dparser
+master_sites        https://github.com/jplevyak/dparser/releases/download/v${version}
```

Comment:
Yah, and the github portgroup handles this, too. I suggest that you fix this before I merge it. Sorry for the delays.